pytket/binders/include/UnitRegister.hpp

```diff
@@ -13,11 +13,12 @@
 // limitations under the License.
 
 #pragma once
+#define _TKET_REG_WIDTH 32
 
 #include "Utils/UnitID.hpp"
 
 namespace tket {
-
+typedef uint32_t _tket_uint_t;
```

Comment:
Minor point, but since any changes to this `typedef` and the `#define` must be synchronized, it would be better if they were on adjacent lines, perhaps even with a comment.